### PR TITLE
check availability and peer readiness before failover/relocate

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://bugzilla.redhat.com/show_bug.cgi?id=2141001

### CODE REVIEW

File: packages/mco/components/disaster-recovery/app-failover-relocate/subscription-group-selector.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ const getOptions = (
       </HelperTextItem>
       <HelperTextItem variant="indeterminate">
         <i>
-          {isPeerReady(drPolicyControlState?.drPolicyControl)
+          {isPeerReadyAndAvailable(drPolicyControlState?.drPolicyControl)
             ? t('Peer ready')
             : t('Peer not ready')}
```
Comment (SanjalKatiyar) [inline packages/mco/components/disaster-recovery/app-failover-relocate/subscription-group-selector.tsx:86]: I don't think we need to fix "Peer ready" and "Peer not ready"... current text looks okay...
ramen gonna fix api anyway, availability check is only for safety. Let me know what u think about this @GowthamShanmugam

Comment (GowthamShanmugam) [reply packages/mco/components/disaster-recovery/app-failover-relocate/subscription-group-selector.tsx:86]: Yes, this is fine

**Review by GowthamShanmugam — APPROVED** [review]

**Review by bipuladh — APPROVED** [review]

**Pull request merged**